src/GenX.jl

```diff
@@ -39,7 +39,7 @@ using Statistics
 
 # Uncomment if Gurobi or CPLEX active license and installations are there and the user intends to use either of them
 #using CPLEX
-#using Gurobi
+using Gurobi
```

Comment:
Please remove this line - it'll cause trouble for anyone trying to build. Same with the add to Project.toml.